exercises/practice/rail-fence-cipher/rail-fence-cipher.spec.wren

```diff
@@ -0,0 +1,32 @@
+import "./rail-fence-cipher" for RailFenceCipher
+import "wren-testie/testie" for Testie, Expect
+
+Testie.test("RailFenceCipher") { |do, skip|
+  do.describe("encode") {
+    do.test("encode with two rails") {
+      Expect.value(RailFenceCipher.encode("XOXOXOXOXOXOXOXOXO", 2)).toEqual("XXXXXXXXXOOOOOOOOO")
+    }
+
+    skip.test("encode with three rails") {
+      Expect.value(RailFenceCipher.encode("WEAREDISCOVEREDFLEEATONCE", 3)).toEqual("WECRLTEERDSOEEFEAOCAIVDEN")
+    }
+
+    skip.test("encode with ending in the middle") {
+      Expect.value(RailFenceCipher.encode("EXERCISES", 4)).toEqual("ESXIEECSR")
+    }
+  }
+
+  do.describe("decode") {
+    skip.test("decode with three rails") {
+      Expect.value(RailFenceCipher.decode("TEITELHDVLSNHDTISEIIEA", 3)).toEqual("THEDEVILISINTHEDETAILS")
+    }
+
+    skip.test("decode with five rails") {
+      Expect.value(RailFenceCipher.decode("EIEXMSMESAORIWSCE", 5)).toEqual("EXERCISMISAWESOME")
+    }
+
+    skip.test("decode with six rails") {
+      Expect.value(RailFenceCipher.decode("133714114238148966225439541018335470986172518171757571896261", 6)).toEqual("112358132134558914423337761098715972584418167651094617711286")
```

Comment:
```suggestion
      var expected = "112358132134558914423337761098715972584418167651094617711286"
      var input = "133714114238148966225439541018335470986172518171757571896261"
      var actual = RailFenceCipher.decode(input, 6)
      Expect.value(actual).toEqual(expected)
```
For very long lines, break it up into pieces please.

Reply:
Updated so plaintext and ciphertext are on adjacent lines, and start in the same column position.

I have also updated the rotational-cipher [PR](https://github.com/exercism/wren/pull/152) and crypto-square [PR](https://github.com/exercism/wren/pull/153).